shared/package.json

```diff
@@ -2,8 +2,9 @@
   "name": "alacrity-shared",
   "version": "1.0.0",
   "description": "",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "commonjs",
```

Comment:
it was this?!

Reply:
i think it was this! it allows the .js files to be interpreted as commonjs modules i think?